### PR TITLE
Update to version 1.3

### PR DESCRIPTION
**Pull request by EvilFreelancer**
# List of changed in version 1.3

## Deprecated methods was removed

Here is the list:
* write
* w
* wr
* wri

Instead of these methods now need to use
* query
* q
* qr
* qri

Usage of deprecated methods was also removed from all examples.

## Laravel Facade method \RouterOS::getClient() renamed to \RouterOS::client()

Instead of `\RouterOS::getClient()` need to use `\RouterOS::client()`, plus was added `\RouterOS::config()` which contains all settings of installation provided by Laravel configs.

## /export command support added

As described in issue #32 was not possible to export configuration from RouterOS, but from 1.3 it will be possbie.

Solution is based on [divineomega/php-ssh-connection](https://github.com/DivineOmega/php-ssh-connection) package.

Export can be triggered in two ways, details is [here](https://github.com/EvilFreelancer/routeros-api-php/blob/development/examples/export.php).

## Laravel classes renamed

Instead of long class names, abbreviated names are now used, this makes the code a little more readable.

Tests show that this does not lead to problems, since the service is registered automatically.

## Other minor changes

* Documentation updated, added information about:
  * More examples and notes about `equal` method
  * Known issues (which in general issues from RouterOS side)
  * information about `/export` and `->export()` method
* Code cleanup, non-used code removed, documentation code refactored
* [larapack/dd](https://github.com/larapack/dd) added to require-dev packages list of `composer.json`
* Finetune of `preconf.tcl` script, colors disabled, now script work faster
* `phpunit/phpunit` version changed from 7.0 to 8.0
* Additional visualization of tests with help of `limedeck/phpunit-detailed-printer` package
* Names of some tests was changed

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,10 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="vendor/autoload.php"
+         colors="true"
```
Comment (Compolomus) [inline phpunit.xml:5]: without colors less informative

File: tests/ConfigTest.php [hunk 2]
```diff
@@ -8,58 +8,58 @@
...
         $obj    = new Config();
         $params = $obj->getParameters();
 
         $this->assertCount(5, $params);
-        $this->assertEquals($params['legacy'], false);
-        $this->assertEquals($params['ssl'], false);
-        $this->assertEquals($params['timeout'], 10);
-        $this->assertEquals($params['attempts'], 10);
-        $this->assertEquals($params['delay'], 1);
+        $this->assertEquals(false, $params['legacy']);
```
[the remark this answers is not in the document]
Comment (EvilFreelancer) [reply tests/ConfigTest.php:27]: @deprecated https://github.com/sebastianbergmann/phpunit/issues/3494

File: src/ResponseIterator.php [hunk 3]
```diff
@@ -2,10 +2,15 @@
 
 namespace RouterOS;
 
-use \Iterator,
-    \ArrayAccess,
-    \Countable,
-    \Serializable;
+use \Iterator;
```
[the remark this answers is not in the document]
Comment (EvilFreelancer) [reply src/ResponseIterator.php:5]: fixed

File: phpunit.xml (hunk 1 above)
Comment (EvilFreelancer) [reply phpunit.xml:5]: yeah, pretty awesome thing :)

File: composer.json [hunk 4]
```diff
@@ -47,6 +47,16 @@
   "require-dev": {
     "limedeck/phpunit-detailed-printer": "^5.0",
     "orchestra/testbench": "^4.0|^5.0",
-    "phpunit/phpunit": "^8.0"
+    "phpunit/phpunit": "^8.0",
+    "roave/security-advisories": "dev-master",
+    "squizlabs/php_codesniffer": "^3.5"
+  },
+  "scripts": {
```
[the remark this answers is not in the document]
Comment (EvilFreelancer) [reply composer.json:56]: yep, additional notes about installed packages, but probably need to remove it, because this project is just a library

**Pull request merged**